### ISSUE TITLE
no-test-support-import false positives

### ISSUE THREAD

**Issue by EvgenyOrekhov** [ember-cli/eslint-plugin-ember#970]
C:\app\tests\test-helper.js

```js
import { setApplication } from "@ember/test-helpers";
import start from "ember-exam/test-support/start";
```

```text
C:\app\tests\test-helper.js
  2:1  error  Do not import a file from test-support into production code, only into test files  ember/no-test-support-import
```

C:\app\tests\unit\controllers\reseller\dashboard-test.js

```js
import { module, test } from "qunit";
import { setupTest } from "ember-qunit";
import window from "ember-window-mock";
import { setupWindowMock } from "ember-window-mock/test-support";
```

```text
C:\app\tests\unit\controllers\reseller\dashboard-test.js
  4:1  error  Do not import a file from test-support into production code, only into test files  ember/no-test-support-import
```
